iosApp/Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift

```diff
@@ -110,7 +114,13 @@ struct SurveyDetailView: View {
                 .accessibility(.surveyDetail(.startButton))
             } else {
                 Button {
-                    // TODO: Add action when press next
+                    // TODO: Submit button logics
+                    let totalItem = (dataSource.viewState.surveyDetail?.questions.count ?? 0) - 1
+                    guard questionIndex < totalItem
+                    else { return }
```

Comment:
Looks like we can shorten this line a bit 
```suggestion
                    guard questionIndex < totalItem else { return }
```